dags/dbt_weekly.py

```diff
@@ -38,7 +38,7 @@ def params_check(params=None, **kwargs):
             kwargs["ti"].xcom_push("dbt_run_args", "--full-refresh --exclude marts.daily marts.oneshot marts.manual")
         else:
             kwargs["ti"].xcom_push("dbt_seed_args", "")
-            kwargs["ti"].xcom_push("dbt_run_args", "--select marts.weekly legacy.weekly ephemeral indexed staging")
+            kwargs["ti"].xcom_push("dbt_run_args", "--select +marts.weekly+ +marts.legacy+")
```

Comment:
`marts.legacy` n'existe pas, non ? 🤔
```suggestion
            kwargs["ti"].xcom_push("dbt_run_args", "--select +marts.weekly+ +legacy.weekly+")
```

Reply:
c'est ça de pousser des PRs en reunion lol